waiter/src/waiter/util/utils.clj

```diff
@@ -1,12 +1,17 @@
 ;;
-;;       Copyright (c) 2017 Two Sigma Investments, LP.
-;;       All Rights Reserved
+;; Copyright (c) Two Sigma Open Source, LLC
```

Comment:
This should go into a separate PR :(

Reply:
I thought the copyright header was already updated in all of our source files, and that I reverted all of the changes to utils.clj! Not sure what happened here. I'll fix it though.